### PR TITLE
Modify getting started tutorial -- now with the correct commits!

### PR DESCRIPTION
**Pull request by aofarrel**
This is the same as #163 but it doesn't have the extra commits from develop added by mistake. I had previously based that branch on develop, then changed the branch I was merging into, so it was picking up a lot of stray commits.

### CODE REVIEW

**Review by denis-yuen — APPROVED** [review]

**Review by NatalieEO — APPROVED** [review]

File: docs/advanced-topics/tools-vs-workflows.rst [hunk 1]
```diff
@@ -0,0 +1,117 @@
+Tools vs Workflows: Detailed Comparison
```
Comment (coverbeck) [inline docs/advanced-topics/tools-vs-workflows.rst:1]: This is the only colon in the sidebar. Suggest removing, which may (or may not) require some rewording

File: docs/advanced-topics/tools-vs-workflows.rst [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
++------------------------+------------------------------------------+-------------------------------------------------+
+
+What about workflows that run in Docker containers?
+---------------------------------------------------
+
+Workflow descriptor files do not require a Docker container. That being said, you can still specify external Docker images
+within the descriptor files, and doing so is generally recommended. Each task (WDL) or step (CWL) can specify a Docker image, which might be totally different from the Docker image used by a previous task/step in the workflow. Because of this, workflows registered on Dockstore that reference Docker image(s) will still follow versioning from GitHub, instead of versioning based on the tags of the Docker image(s).
+
+.. hint::
+    The Cromwell execution engine requires that WDL task specify a Docker image when running on GCP or AWS backends, including Terra. Therefore, if your WDL workflow needs to run on Terra, make sure that each task uses the Docker runtime attribute. For more information see the `Cromwell docs <https://cromwell.readthedocs.io/en/stable/RuntimeAttributes/#docker>`_.
```
Comment (coverbeck) [inline docs/advanced-topics/tools-vs-workflows.rst:59]: "that WDL task specify..." -> "that a WDL task specify..." or "that WDL tasks specify..."

Comment (coverbeck) [inline docs/advanced-topics/tools-vs-workflows.rst:59]: > WDL task specify a Docker image when running on GCP or AWS backends

Do you mean if you're running Cromwell without Terra? Sounds like... I'm not sure in all cases Cromwell would know it's running on AWS (you're just on EC2 instance), but probably not worth getting into that.

File: docs/getting-started/intro-to-dockstore-tools-and-workflows.rst [hunk 3]
```diff
@@ -1,98 +1,32 @@
...
-However, our definition for each does not completely align with the language's specification.
-Instead, Dockstore tools are more associated with creating/owning Docker images that are used in conjunction with a descriptor language, and
-Dockstore workflows are more closely tied to the descriptor files themselves. While other descriptor languages, like WDL and Nextflow,
-do not have separate concepts for tools and workflows, we still maintain a distinction between a tool and a workflow for WDL.
+vcf file --> convert to GDS file --> give each variant a unique ID --> a GDS file with unique IDs
 
+In this case, the output of the conversion step is the input of the step that generates unique IDs, and the unique IDs step's output is the final output of the workflow: A GDS file with unique IDs. 
 
+Managing workflows can be complicated, as you may be chaining together many steps, each with different computational requirements. This where workflow languages step in, allowing us to create formal "tasks" which can each have their own computational settings or Docker container, and chaining the output of one task into the input of another task.
```
Comment (coverbeck) [inline docs/getting-started/intro-to-dockstore-tools-and-workflows.rst:13]: > This where workflow

This is where workflow...

**Review by coverbeck — APPROVED** [review]

File: docs/advanced-topics/tools-vs-workflows.rst (hunk 2 above)
Comment (aofarrel) [reply docs/advanced-topics/tools-vs-workflows.rst:59]: Strictly speaking, [Cromwell's docs say this about the docker runtime attribute](https://cromwell.readthedocs.io/en/stable/RuntimeAttributes/#docker):

> GCP: This attribute is mandatory when submitting tasks to Google Cloud.
> AWS Batch: This attribute is mandatory when submitting tasks to AWS Batch.

Due to how the docs tend to present things as "this is what changes based on the backend," I interpreted this to mean "any AWS backend," but I will be the first to admit I know very little about AWS. What's your interpretation?

Comment (coverbeck) [reply docs/advanced-topics/tools-vs-workflows.rst:59]: My interpretation is that if you use Cromwell with AWS Batch, which will let Cromwell scale up on its own, then you need it.

But if you just ssh into a machine that happens to be hosted on AWS and running Cromwell locally, it doesn't matter.

Same is probably true for GCP...

So, I would either drop the the Cromwell execution engine reference, or if you want to refer to it, maybe talk about in terms of [scaling](https://cromwell.readthedocs.io/en/stable/Scaling/).

Comment (aofarrel) [reply docs/advanced-topics/tools-vs-workflows.rst:59]: Good point -- I clarified it.

**Pull request merged**